### PR TITLE
[BPK-3546] Bridge calendar breaking changes

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-react-native/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] `UNRELEASED.md`
+ [ ] `README.md`
+ [ ] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendarSelectionConfigurationConstants.m [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+
+#import "RCTBPKCalendarSelectionConfigurationConstants.h"
+
+#import <Backpack/Calendar.h>
+
+@implementation RCTBPKCalendarSelectionConfigurationConstants
+
++ (BPKCalendarSelectionConfiguration *)single {
+    return [BPKCalendarSelectionConfigurationSingle new];
```
Comment (georgegillams) [inline lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendarSelectionConfigurationConstants.m:26]: As react only sends changes in props, we should find that new instances are only changed when the selection type prop changes

File: lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendar.m [hunk 2]
```diff
@@ -67,9 +67,10 @@ - (void)setupCalendar {
     self.utcCalendar.timeZone = [[NSTimeZone alloc] initWithName:@"UTC"];
 }
 
-- (void)setRct_selectionType:(BPKCalendarSelection)rct_selectionType {
-    if(_rct_selectionType != rct_selectionType) {
-        _rct_selectionType = rct_selectionType;
+- (void)setRct_selectionConfiguration:(BPKCalendarSelectionConfiguration *)rct_selectionConfiguration {
+    // Object comparison will always be false as we're using new instances each time?
```
Comment (k0nserv) [inline lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendar.m:71]: Worth verifying this

**Review by k0nserv — APPROVED** [review]
LGTM

File: lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendar.m (hunk 2 above)
Comment (georgegillams) [reply lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendar.m]: This was actually a note to myself to remind myself to remove it 😆

**Pull request merged**